src/ansible_creator/subcommands/init.py

```diff
@@ -61,10 +60,7 @@ def run(self: Init) -> None:
             self.init_exists()
         self._init_path.mkdir(parents=True, exist_ok=True)
 
-        if self._project == "collection":
```

Comment:
I kind of like this dispatch here, can we call the same function and pass the message to it?

Reply:
I think it suggests how to handle the next one and not have project specific conditionals in the scaffold def

Reply:
If by message, you mean the string passed to `self.output.note()` at the end, at that point we might as well take the call to note out of the scaffold method, I guess?

But really the issue is that the output is insufficiently similar, yes? I concocted this to match the tests we have, but is there rather a way to make the output scan both ways while providing _project as a variable?